lib/rules/no-unbalanced-curlies.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const Rule = require('./base');
+
+const ERROR_MESSAGE = 'Unbalanced curlies detected';
+const DISALLOWED_CHARS = '}}';
+
+module.exports = class NoUnbalancedCurlies extends Rule {
+  visitor() {
+    return {
+      TextNode(node) {
+        let chars = node.chars;
+        if (chars.includes(DISALLOWED_CHARS)) {
+          let loc = node.loc;
+          this.log({
+            message: ERROR_MESSAGE,
+            line: loc && loc.start.line,
+            column: loc && loc.start.column + chars.indexOf(DISALLOWED_CHARS),
```

Comment:
I'm not sure if this work, if the `chars` contain line breaks 🤔 

could you add a test case for that?

Reply:
Good catch; I added a few cases. I wasn't sure how much to test. I may have overdone it (or underdone?). Incidentally, I took the regexp from `lib/rules/linebreak-style.js`. Anyway, some new questions:

1. My implementation allows for more than one warning, as the logger is in the loop. I couldn't quite see how to write the test for this. Did I miss an example somewhere? Should I bother?
2. Should some code be refactored and DRY'd between `lib/rules/linebreak-style.js` and this?
3. Should `source` (in `this.log()`) be only the affected line, or should it be the whole node (as it is now)?
4. Also reading from `lib/rules/linebreak-style.js`, I see it searches from line breaks within `this.sourceForNode(node)`, while I'm looking in `chars`. What's the difference? Should I do the same?

And probably others I'm missing now!